src/wazuh_db/wdb_parser.c

```diff
@@ -5971,7 +5974,9 @@ int wdb_parse_agents_insert_vuln_cves(wdb_t* wdb, char* input, char* output) {
         }
         else {
             cJSON* result = wdb_agents_insert_vuln_cves(wdb, j_name->valuestring, j_version->valuestring, j_architecture->valuestring, j_cve->valuestring,
-                                            j_reference->valuestring, j_type->valuestring, j_status->valuestring, (bool)j_check_pkg_existence->valueint);
+                                            j_reference->valuestring, j_type->valuestring, j_status->valuestring, (bool)j_check_pkg_existence->valueint,
```

Comment:
Wouldn't be nice to unify this, using cJSON_GetStringValue for all the previous items? Also check the indentation.

Reply:
Fixed!